### PR TITLE
Fixes motion sensitive cameras

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do
Fixes motion cameras, of which outside motion sensitive areas (EVA) have been broken since #14196
Motion sensitive cameras now work off the cameras view range outside motion sensitive areas, instead of *one tile*
Bots no longer set off motion alarms, so the armory doesn't ping a motion alarm constantly.
Fixes motion cameras breaking if the last mob leaves.
makes motion sensitive cameras mention the area vs the camera.

## Why It's Good For The Game
Motion sensitive cameras working are good. Means armory breaches / ai sat breaches (not vault, vault has no motion sensitive camera) are detected faster, if the antagonist does not cut the camera in time, the AI will notice.
Bots no longer set off motion alarms so the armory doesn't constantly go off.
The motion range is now the cameras range in tiles instead of *one tile* However, if you change the range of a cameras view, it will detect from farther away. This might actually lead to it being of use.

## Testing

Tested the camera areas (EVA,  worked)
Tested a 7 tile camera, worked, disabled after mob left.
Tested a 4 tile, only worked in 4 tiles, disabled after.
Tested bots did not set off the motion sensor.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

image

sorry for balance in a fix but 1 tile would like never trigger

## Changelog

:cl:
fix: Fixes motion cameras not working always outside the eva room
tweak: Motion sensitive cameras outside EVA have a detection range equal to their camera view, rather than *one  tile*
/:cl:

### CODE REVIEW

File: code/game/machinery/camera/motion.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	else if(detectTime == -1)
 		for(var/thing in getTargetList())
 			var/mob/target = locateUID(thing)
-			if(QDELETED(target) || target.stat == DEAD || (!area_motion && !in_range(src, target)))
+			if(QDELETED(target) || target.stat == DEAD || (!area_motion && (!(get_dist(src, target) <= view_range) || !(locate(target) in hearers(view_range, get_turf(src))))))
```
[the remark this answers is not in the document]
Comment (Qwertytoforty) [reply code/game/machinery/camera/motion.dm:15]: testing

[the remark this answers is not in the document]
Comment (Qwertytoforty) [reply code/game/machinery/camera/motion.dm:15]: suffering indeed

[the remark this answers is not in the document]
Comment (Qwertytoforty) [reply code/game/machinery/camera/motion.dm]: done!

File: code/game/machinery/camera/motion.dm [hunk 2]
```diff
@@ -24,6 +24,10 @@
 /obj/machinery/camera/proc/newTarget(mob/target)
 	if(isAI(target))
 		return FALSE
+	if(isbot(target)) //No armsky, you don't get to set off the motion alarm constantly
+		return FALSE
+	if(!area_motion && (!(get_dist(src, target) <= view_range) || !(locate(target) in hearers(view_range, get_turf(src)))))
```
[the remark this answers is not in the document]
Comment (Qwertytoforty) [reply code/game/machinery/camera/motion.dm]: Done!

**Review by DGamerL — APPROVED** [review]
LGTM

**Pull request merged**